scripts/SetupServiceFabric/SetupServiceFabric.sh

```diff
@@ -5,6 +5,11 @@
 # It also sets up Azure Service Fabric CLI
 #
 # Usage: sudo ./SetupServiceFabric.sh
+# Setting up Service Fabric from local service fabric runtime and sdk debain packages is all supported, both two debain packages will be needed for installation.
```

Comment:
"sdk debain packages is all supported" - Fix

Reply:
Fixed.